includes/cli/class-import.php

```diff
@@ -45,7 +48,16 @@ class Import extends WP_CLI_Command {
 	public function __invoke( $args, $assoc_args ) {
 		$ignore_terms = WP_CLI\Utils\get_flag_value( $assoc_args, 'ignore-terms', false );
 		$create_terms = WP_CLI\Utils\get_flag_value( $assoc_args, 'create-terms', false );
-		$file         = $args[0];
+		$ras_defaults = WP_CLI\Utils\get_flag_value( $assoc_args, 'ras-defaults', false );
+		$file         = $args[0] ?? null;
+
+		if ( ! $file && ! $ras_defaults ) {
```

Comment:
Suggestion – moving this check below the `$file` assignment below would make this `$file` check more universal. `$file` would be either user-supplied JSON or a preset, in case more presets are added in the future.

Reply:
done

Reply:
What I meant was something different – since `$file` may be set on line 55, later you can just do:

```
	if ( ! $file ) {
			WP_CLI::error( __( 'You must either supply a json file path or use a preset flag.', 'newspack-popups' ) );
		}
```

But it's a nitpick!

Reply:
ah of course